packages/webapp/src/containers/Animals/Inventory/index.tsx

```diff
@@ -102,9 +105,18 @@ function AnimalInventory({
     [updateSelectedTypeIds],
   );
 
+  const onRemovalSuccess = (animalOrBatchKey: AnimalOrBatchKeys, ids: Animal['id'][]) => {
+    setSelectedInventoryIds((selectedInventoryIds) => {
+      return selectedInventoryIds.filter((i) => {
+        const { kind, id } = parseInventoryId(i);
+        return kind === animalOrBatchKey && ids.includes(id);
```

Comment:
The previous filter was : `!selectedAnimalIds.includes(i))` 

Everything looks like it works so I am not sure it matters, but it looks like it is setting the selected inventory id to the ones that were successfully removed?

Reply:
Thank you so much for catching it! I remember getting confused about how to use `.filter()` suddenly and ended up flipping the condition! 🙏
It's fixed now!